PostHog/posthog
# Warn when dashboard is available

## Changes

It's confusing when you get sent a link to a dashboard but you've selected a different project, as you'll get a "dashboard not found" message. This PR allows you to easily switch to the correct project.

![image](https://user-images.githubusercontent.com/1727427/150994804-5eacea3e-498f-448d-9f2f-457842e91d71.png)


## How did you test this code?



CI + manual QA:
- Have 2 projects
- Copy the link of a dashboard in project 1 and switch to project 2
- Load the URL of project one, see the warning appear
